Application/EdFi.Admin.DataAccess/Contexts/UsersContext.cs

```diff
@@ -19,7 +19,8 @@ public abstract class UsersContext : DbContext, IUsersContext
         protected UsersContext(string connectionString)
             : base(connectionString)
         {
-            Database.SetInitializer(new ValidateDatabase<UsersContext>());
+            Database.SetInitializer(new ValidateDatabase<SqlServerUsersContext>());
```

Comment:
The current version of EF requires the concrete implementation of the Initializer context, note that the initializers weren't being executed because of this, so EF fell back to the default behavior of creating the database if it didn't exist; and it failed to do so because of a cascade delete validation.

This made the error message misleading; the users didn't know that the actual error was an invalid database name in the connection string.